tools/serve/serve.py

```diff
@@ -366,6 +366,7 @@ def add_mount_point(self, url_base, path):
             ("GET", "*.any.serviceworker.html", ServiceWorkersHandler),
             ("GET", "*.any.worker.js", AnyWorkerHandler),
             ("GET", "*.asis", handlers.AsIsHandler),
+            ("GET", "/.well-known/origin-policy", handlers.PythonScriptHandler),
```

Comment:
Have you verified this works? I don't know how helpful it would be, but we could write a custom handler here so that each well-known domain gets its own specific Python file as the handler (which could live where ever we want), rather than having a single Python function to handle all subdomains.

Reply:
Yep! This is working great in https://github.com/web-platform-tests/wpt/pull/21739.

Reply:
OK. For my money the code in the `.well-known/origin-policy` file would be better in a handler, but I don't object to this approach.